packages/core-web/webpack.dev.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
is this due to `require`?

Reply:
let's use `/* eslint-env node */` here instead https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
Hi Zeyu! It is due to module.exports and __dirname.